transifex/pull.py

```diff
@@ -218,11 +226,16 @@ def pull(repo):
 
 
 if __name__ == '__main__':
-    settings_file = join(abspath(dirname(__file__)), 'settings.yaml')
-    with open(settings_file) as f:
-        settings = yaml.load(f)
-
-    with concurrent.futures.ProcessPoolExecutor() as executor:
-        for clone_url in settings['repos']:
-            repo = Repo(clone_url)
-            executor.submit(pull, repo)
+    try:
+        clone_url = sys.argv[1]
```

Comment:
I'll go with your suggestion below. It's not necessary to have both messages. The absence of one implies the other.